### PR TITLE
Link database migration documentation from Galaxy docs

### PR DESCRIPTION
**Pull request by jdavcs**
Follow-up to review of #14537.

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [x] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

### CODE REVIEW

File: doc/source/dev/faq.rst [hunk 1]
```diff
@@ -13,6 +13,12 @@ This can be done with either IPython or a plain Python console, depending on you
 
     python -i scripts/db_shell.py
 
+
+... upgrade, downgrade, or modify the Galaxy database schema?
+-------------------------------------------------------------
+
+Please read Galaxy's [database schema migration documentation](https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/model/migrations/README.md).
```
Comment (mvdbeek) [inline doc/source/dev/faq.rst]: Can you please link the document itself up ? In the admin section.

**Comment by jdavcs** [comment]
I have added this to Galaxy documentation. Once we publish the next release, I'll replace the content of `README.md` with a link to the documentation site.
I've placed the section between Conda and Reports in the main Administration section, but it can go anywhere else.

I've also fixed a minor typo in the Config section - "Config Options" now shows up as a subheading on the main Admin docs page.

**Comment by nsoranzo** [comment]
> Once we publish the next release, I'll replace the content of README.md with a link to the documentation site.

Alternatively, you can target this PR to 22.05.

**Comment by jdavcs** [comment]
> > Once we publish the next release, I'll replace the content of README.md with a link to the documentation site.
> 
> Alternatively, you can target this PR to 22.05.

Obviously! Thanks :) Retargeting..

**Comment by jdavcs** [comment]
Replaced by #14715

**Pull request closed without merging**